src/app/view/components/_common/dot-add-to-bundle/dot-add-to-bundle.component.html

```diff
@@ -14,7 +14,7 @@
 
     <form [formGroup]="form" (ngSubmit)="submitBundle($event)" (keyup.enter)="submitBundle($event)" novalidate #formEl="ngForm">
         <div class="form__group">
-            <p-dropdown #addBundleDropdown appendTo="body" [options]="bundle$ | async" editable="true" [autoWidth]="false" formControlName="addBundle" [placeholder]="placeholder" optionLabel="name" ></p-dropdown>
+            <p-dropdown #addBundleDropdown appendTo="body" [options]="bundles" editable="true" [autoWidth]="false" formControlName="addBundle" [placeholder]="placeholder" optionLabel="name"  ></p-dropdown>
```

Comment:
Why?

Reply:
we used to have two request to `getBundles`. One is to define the placeholder so there I can assign the bundles instead make other request.